discord_bot/discord_bot.py

```
@@ -113,17 +116,98 @@ async def send_notice(notice: NoticeData, scraper_type: ScraperType):
+                            f"채널 [{channel_name}]에 공지사항을 전송했습니다: {notice.title}"
+                        )
+                    else:
+                        logger.error(
+                            f"채널 {channel_id}: 알 수 없는 채널 타입 {type(channel)}"
+                        )
+
+            except discord.Forbidden as e:
+                if "channel" in locals() and isinstance(channel, discord.DMChannel):
+                    logger.error(
+                        f"[{scraper_type.korean_name}] {channel_id} 유저: {channel.recipient} 에 메세지를 보낼 권한이 없습니다."
+                    )
```

Ruff (0.14.8)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)